provd/internal/services/manager_test.go

```diff
@@ -0,0 +1,71 @@
+package services_test
+
+import (
+	"context"
+	"flag"
+	"testing"
+
+	"github.com/canonical/ubuntu-desktop-provision/provd/internal/services"
+	"github.com/canonical/ubuntu-desktop-provision/provd/internal/testutils"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+)
+
+func TestNewManager(t *testing.T) {
+	tests := map[string]struct {
+		wantErr bool
+	}{
+		"Successfully create the manager": {},
```

Comment:
created* is probably the best :slightly_smiling_face:

Reply:
Actually, this is an action describe that with this input, it will create the manager with success, so present simple (conjugated) is better? It’s a little bit like the documentation for functions/methods.

(This is what we did until then in other projectrs, but happy to be corrected by a native speaker :))

Reply:
Ah, sorry, misread the context! creates* is correct :smile: